src/components/AttentionBox/AttentionBox.jsx

```diff
@@ -13,6 +13,8 @@ const AttentionBox = ({
   className,
   // Backward compatibility for props naming
   componentClassName,
+  // Will remove when releasing version 2 as BREAKING CHANGES
+  withIconWithoutHeader,
```

Comment:
because until now we did not support displaying icon without title.
we still don't support it in regular mode but in compact mode it is looking good and needed. (example in screenshot in the pr)

Reply:
that is a breaking change and that's why we add the flag